modules/secret-manager/README.md

```diff
@@ -33,8 +37,12 @@ module "secret-manager" {
   source     = "./fabric/modules/secret-manager"
   project_id = "my-project"
   secrets = {
-    test-auto   = null
-    test-manual = ["europe-west1", "europe-west4"]
+    test-auto = {
```

Comment:
I think we can shorten this example to:
```hcl
test-auto = {}
```

Reply:
With the last commit I also introduced the default value on the location, so as to allow the module to be used as you suggested.